CHANGELOG.md

```diff
@@ -34,6 +34,11 @@ Dropping a requirement of a major version of a dependency is a new contract.
 
 [JPREF-490]: https://ecosystem.atlassian.net/browse/JPERF-490
 
+### Added
+- Add `NonparametricFacade`, `RelativeNonparametricPerformanceJudge` and `RelativePerformanceStabilityJudge` which are necessary for [JPERF-493].
```

Comment:
I like that we're exposing the sub-judges separately. This way we can let users try them out and if they work well for enough use cases and for a long enough time, we can attach them to `BaselineComparingJudge`.